README.md

```diff
@@ -1 +1,164 @@
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#Run-tests)
+- [👥 Authors](#authors)
+- [🎥 Live Demo](#live-demo)
+- [🎥 Project Presentation](#Project-presentation)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgments](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 My Portfolio <a name="about-project"></a>
+
+> this Project shows the previous 4 blocks of current module's effort and I have developed and deployed on GIthub, showcasing my skills in HTML and CSS with JS.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">Html</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/css/default.asp">css</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Use the semantics elements💯
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A web browser to view output e.g [Google Chrome](https://www.google.com/chrome/).
+- An IDE e.g [Visual studio code](https://code.visualstudio.com/).
+- Install the `npm` package manager use this [to install both node and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
+- [A terminal](https://code.visualstudio.com/docs/terminal/basics).
+
+### Setup
+
+Clone this repository to your desired folder or download the Zip folder:
+
+```
+git clone https://github.com/BilalLiaquat7/module1_capstone.git
+```
+
+- Navigate to the location of the folder in your machine:
+
+**`you@your-Pc-name:~$ cd desktop`**
+
+### Install
+
+To install all dependencies, run:
+
+```
+npm install
+```
+
+### Usage
+
+To run the project, follow these instructions:
+
+- After Cloning this repo to your local machine.
+- To get it running on your default browser and local host, run:
+
+1. Open the index with live server from the code editor
+2. Or you can double click on the index to open it in the browser
+
+### Run tests
+
+To run tests, run the following command:
+
+- Track HTML linter errors run:
+
+```
+npx hint .
+```
+
+- Track CSS linter errors run:
+
+```
+npx stylelint "**/*.{css,scss}"
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Muhammad Bilal Liaquat**
+
+- GitHub: [@githubhandle](https://github.com/BilalLiaquat7/)
+- LinkedIn: [@Linkedin](https://www.linkedin.com/in/muhammad-bilal-liaquat-87863390/)
+
+<p align="right">(<a href="#live-demo">back to top</a>)</p>
+
+## 🎥 Live Demo <a name="authors"></a>
+
+- Live Demo: [@LiveDemo](https://bilalliaquat7.github.io/module1_capstone/)
```

Comment:
- [ ] As the previous reviewer mentioned your live demo is not showing the website kindly make sure to deploy this branch for now instead of the `main` branch so the live demo is working as expected